addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -756,9 +761,9 @@
         </t>
         <t t-else="">
             <a t-if="can_access" t-attf-href="/slides/slide/#{slug(slide)}#{query_string}" t-title="slide.name">
-                <div class="card-img-top border-bottom o_wslides_gradient" t-attf-style="padding-top: 50%;"/>
+                <div class="o_wslides_body card-img-top border-bottom" t-attf-style="padding-top: 50%;"/>
```

Comment:
"o_wslides_body" is generally used as a top class that wraps the whole thing.

Here you used it for smaller bits of inner HTML, maybe we should make a different class in that case?
Not sure what rules you are using, probably the background-color?